src/AutoSplit.py

```diff
@@ -283,7 +283,8 @@ def __reload_start_image(self, started_by_button: bool = False, wait_for_delay:
         self.timer_start_image.stop()
         self.current_image_file_label.setText("-")
         self.start_image_status_value_label.setText("not found")
-        set_preview_image(self.current_split_image, None)
+        if not self.current_split_image.text:
+            set_preview_image(self.current_split_image, None)
```

Comment:
If we're looking for a certain text instead of an image, it'd be better to display said text in the image preview

Reply:
Done.